src/core/meshcop/joiner.hpp

```diff
@@ -109,12 +109,28 @@ class Joiner : public InstanceLocator
      */
     void GetJoinerId(Mac::ExtAddress &aJoinerId) const;
 
+    /**
+     * This method validates the PSKd.
+     *
+     * Per Thread specification, a Joining Device Credential is encoded as
+     * uppercase alphanumeric characters (base32-thread: 0-9,A-Y excluding
+     * I,O,Q, and Z for readability) with a minimum length of 6 such
```

Comment:
```suggestion
     * uppercase alphanumeric characters (base32-thread: 0-9, A-Z excluding
     * I, O, Q, and Z for readability) with a minimum length of 6 such
```
nit: if we say we are excluding "Z", then we should include it in the range A-Z.

Reply:
Agree.